lib/configuration.rb

```diff
@@ -145,6 +145,15 @@ def self.html_to_pdf_command
     super
   end
 
+  def self.raw_emails_location
+    warn <<~DEPRECATION.squish
+      [DEPRECATION] AlaveteliConfiguration.raw_emails_location will be removed
+      in 0.42. You should have `ActiveStorage` configured but you still haven't
+      migrated all your old files. Please un `rails storage:migrate` to migrate
```

Comment:
`s/un/run/`

Reply:
Need to add `bin/` as per 1591ec4d0fe20b0387d3ea9292de16202dda600f

Reply:
Ugh, fixed in ce379fdc5